enterprise-suite/install-es.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function docvar() {
+    envvar=$1
+    local detail=$2
+    eval echo "$envvar=\$${envvar}"
+    if [ -n "$detail" ]; then
+        echo -e "\t $detail"
+    fi
+    echo
+}
+
+function usage() {
+    echo "$0 [-h]"
+    echo
+    echo "Environment variables that can be set:"
+    echo
+    docvar LIGHTBEND_COMMERCIAL_CREDENTIALS "Credentials file in property format with username/password"
+    docvar ES_REPO "Helm chart repository"
+    docvar ES_CHART "Chart name to install from the repository"
+    docvar ES_NAMESPACE "Namespace to install ES-Console into"
+    docvar ES_MINIKUBE "Set to true to enable minikube specific options"
+    docvar ES_LOCAL_CHART "Set to location of local chart tarball"
+    docvar ES_UPGRADE "Set to true to perform a helm upgrade instead of an install"
+    docvar ES_VERSION "Set to non-empty to install a specific version"
+    exit 1
+}
+
+function export_credentials() {
+    # Exported variables.
```

Comment:
Where are these exported?  Doesn't matter but makes me think about the name/comment.